http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -61,7 +61,7 @@ class AppTable(tag: Tag) extends Table[AppRecord](tag, "APP") {
   def dateAccessed = column[Instant]("dateAccessed", O.SqlType("TIMESTAMP(6)"))
   def namespaceId = column[NamespaceId]("namespaceId", O.Length(254))
   def diskId = column[Option[DiskId]]("diskId", O.Length(254))
-  def customEnvironmentVariables = column[Map[String, String]]("customEnvironmentVariables")
+  def customEnvironmentVariables = column[Option[Map[String, String]]]("customEnvironmentVariables")
```

Comment:
This was actually throwing an error when I deployed it to my fiab:
```
Caused by: slick.SlickException: Read NULL value (null) for ResultSet column <computed>
```
The column needs to be `Option` in slick because it's nullable in mysql.